src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Compiler/FormPassTest.php

```diff
@@ -166,22 +166,36 @@ public function testAddTaggedGuessers()
     /**
      * @dataProvider privateTaggedServicesProvider
      */
-    public function testPrivateTaggedServices($id, $tagName, array $tagAttributes = array())
+    public function testPrivateTaggedServices($id, $tagName, $argumentKey, $expectedArgument, array $tagAttributes = array())
     {
-        $container = $this->createContainerBuilder();
+        $formPass = new FormPass();
+        $container = new ContainerBuilder();
 
         $container->setDefinition('form.extension', $this->createExtensionDefinition());
         $container->register($id, 'stdClass')->setPublic(false)->addTag($tagName, $tagAttributes);
 
-        $container->compile();
+        $formPass->process($container);
+
+        $this->assertEquals($expectedArgument, $container->getDefinition('form.extension')->getArgument($argumentKey));
     }
 
     public function privateTaggedServicesProvider()
     {
         return array(
-            array('my.type', 'form.type'),
-            array('my.type_extension', 'form.type_extension', array('extended_type' => 'Symfony\Component\Form\Extension\Core\Type\FormType')),
-            array('my.guesser', 'form.type_guesser'),
+            array(
+                'my.type',
+                'form.type',
+                0,
+                new Reference('service_locator.c35554e29b2a3001b879847fc6a49848'),
```

Comment:
In #22234, my tests are failing because this id (and the other in `FormPassTest`) is different: https://travis-ci.org/symfony/symfony/jobs/221480972#L2845

Do you have any thoughts? I'm guessing something unrelated I've changed there causes this sha to be different. Is the fix in my PR, or do we need to re-think the test (because these will constantly change)?